src/domains.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from dotenv import load_dotenv
+load_dotenv()
+
+public_address = os.getenv("PUBLIC_ADDRESS")
+private_key = os.getenv("PRIVATE_KEY")
+network = os.getenv("NETWORK")
+
+base_url = f"https://protocol-{network}.gnosis.io/api/v1/"
```

Comment:
A cool new pythonic way of formatting strings!

https://realpython.com/python-f-strings/